File: trunk/unityClientSample/Assets/src/protocol/BattleOverNotify.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
    /// <summary>
    /// 战斗结果 协议:-32
    /// </summary>
	[Proto(value=-32,description="战斗结果")]
	[ProtoContract]
	public class BattleOverNotify
	{
        /// <summary>
        ///  房间id
        /// </summary>
		[ProtoMember(3, IsRequired = false)]
		public string roomId;
        /// <summary>
        ///  胜利true 失败false
        /// </summary>
		[ProtoMember(4, IsRequired = false)]
		public bool flag;
        /// <summary>
        ///  是否成功
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public bool success;
        /// <summary>
        ///  错误消息
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public string info;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/notify/BattleOverNotify.java
/**
 * 资源更新
 */
package org.ngame.protocol.notify;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import lombok.Data;

import org.ngame.protocol.BaseResponse;
import org.ngame.protocol.annotation.Proto;

/**
 *
 * @author csf
 */
@Data
@Proto(value = -32, description = "战斗结果")
public class BattleOverNotify extends BaseResponse
{
   @Protobuf(description = "房间id")
   private String roomId;
   @Protobuf(description = "胜利true 失败false")
   private boolean flag;
}
